File: dsa_js/array_min_element.js
function findMinElement(arr) {
    let minElement = arr[0];
    for(let i = 0; i < arr.length; i++) {
        if (arr[i] < minElement) {
            minElement = arr[i];
        }
    }
    return minElement;
}
const list = [7, 12, 9, 4, 11];
console.log(findMinElement(list));

File: dsa_python/array_min_element.py
def find_min_element(arr):
    min_element = arr[0]
    for i in range(1, len(arr)):
        if arr[i] < min_element:
            min_element = arr[i]
    return min_element

num_list = [7, 12, 9, 4, 11]
print(find_min_element(num_list))